### PR TITLE
Hello World in Kotlin

### PR DESCRIPTION
**Pull request by sirjan13**

### CODE REVIEW

**Comment by sirjan13** [comment]
HelloWorld in Kotlin :)

**Comment by knightking100** [comment]
Invalid

**Comment by knightking100** [comment]
Please read the guide again. Thank you

File: hello.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package hello
```
Comment (knightking100) [inline hello.kt]: Delete this file please

**Pull request closed without merging**